src/test/java/com/karankumar/bookproject/backend/entity/AuthorTest.java

```diff
@@ -94,9 +91,13 @@ void orphanAuthorsRemoved() {
         Book book = new Book("Sophie's World", orphan, toRead);
         bookService.delete(book);
 
-        assertAll(
-                () -> assertThrows(RuntimeException.class, () -> authorService.findById(orphan.getId())),
-                () -> assertTrue(authorService.findAll().isEmpty())
-        );
+        SoftAssertions softly = new SoftAssertions();
+        softly.assertThatThrownBy(
+                () -> authorService.findById(orphan.getId())
+        )
+                .isInstanceOf(RuntimeException.class);
+        softly.assertThat(authorService.findAll()).isEmpty();
+        softly.assertAll();
+
```

Comment:
As per our style guide, could you please refactor this to use `assertSoftly`:
```java
        assertSoftly(
                softly -> {
                    softly.assertThatThrownBy(() -> authorService.findById(orphan.getId()))
                            .isInstanceOf(RuntimeException.class);
                    softly.assertThat(authorService.findAll()).isEmpty();
                }
        );
```

(and add a static import)